ee/clickhouse/views/test/test_clickhouse_action.py

```diff
@@ -36,7 +37,11 @@ def _create_event(**kwargs):
     create_event(**kwargs)
 
 
-class TestAction(
+class TestActionApi(ClickhouseTestMixin, factory_test_action_api(_create_event)):  # type: ignore
+    pass
```

Comment:
is this a no-op? nuke it?

Reply:
It's not. What's happening here is that we're now executing the action api tests against clickhouse installs as well in this file - the inheritance takes care of creating a class containing all the tests.